student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name)
+    super(age, name)
+    @color = classroom
+  end
```

Comment:
- [ ] Good job having the `Student` class extend the `Person` class, however it should also inherit the `parent_permsion`.  Kindly add it also to your constructor.

Reply:
Thanks!